suite-native/atoms/src/Select/Select.tsx

```diff
@@ -24,9 +24,18 @@ type SelectProps = {
     selectLabel: string;
     value: SelectValue;
     onSelectItem: (value: SelectValue) => void;
+    hasBackArrow?: boolean;
+    valueLabel?: string;
 };
 
-export const Select = ({ items, selectLabel, value, onSelectItem }: SelectProps) => {
+export const Select = ({
+    items,
+    selectLabel,
+    value,
+    valueLabel,
+    onSelectItem,
+    hasBackArrow = false,
```

Comment:
So remove this back arrow completely and just keep the close button you think? @dahaca

Reply:
https://github.com/trezor/trezor-suite/pull/5895/commits/285dce9e0a56ad98fd9ab9555c697cffc3a30539

Reply:
Yes, keep the close for now, since it's a different discussion. Although the close button is already missing from the MVP. As you said – it makes sense to concentrate on functionality. since styles and placements are changing often. I will try to comment only on stuff that makes little sense and looks like a bug, rather than an aesthetic mismatch.